authenticate-users/index.php

```diff
@@ -0,0 +1,103 @@
+<?php
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+# [START getting_started_auth_all]
+require_once __DIR__ . '/vendor/autoload.php';
+
+# [START getting_started_auth_certs]
+/**
+ * Returns a dictionary of current Google public key certificates for
+ * validating Google-signed JWTs.
+ */
+function certs() : string
+{
+    $client = new GuzzleHttp\Client();
+    $response = $client->get(
+        'https://www.gstatic.com/iap/verify/public_key-jwk'
+    );
+    return $response->getBody();
+}
+# [END getting_started_auth_certs]
+
+# [START getting_started_auth_audience]
+/**
+ * Returns the audience value (the JWT 'aud' property) for the current
+ * running instance. Since this involves a metadata lookup, the result is
+ * cached when first requested for faster future responses.
+ */
+function audience() : string
+{
+    $metadata = new Google\Cloud\Core\Compute\Metadata();
+    $projectNumber = $metadata->getNumericProjectId();
+    $projectId = $metadata->getProjectId();
+    $audience = sprintf('/projects/%s/apps/%s', $projectNumber, $projectId);
+    return $audience;
+}
+# [END getting_started_auth_audience]
+
+# [START getting_started_auth_validate_assertion]
+/**
+ * Checks that the JWT assertion is valid (properly signed, for the
+ * correct audience) and if so, returns strings for the requesting user's
+ * email and a persistent user ID. If not valid, returns null for each field.
+ *
+ * @param $assertion The JWT string to assert
```

Comment:
These are missing types and the first character of the description should be capitalized